### PR TITLE
Test e2e flakiness

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

# Changes

# Tests

# Todos

- [ ] Add entry to changelog (if necessary).

### CODE REVIEW

File: frontend/src/tests/e2e/accounts.spec.ts [hunk 1]
```diff
@@ -29,7 +29,9 @@ test("Test accounts requirements", async ({ page, context }) => {
   expect(await accountNames()).toEqual([mainAccountName, subAccountName]);
 
   // The linked account should still be present after refresh
-  page.reload();
+  await page.reload();
+  await page.waitForTimeout(5000);
```
Comment (mstrasinskis) [inline frontend/src/tests/e2e/accounts.spec.ts:33]: @peterpeterparker, line 32 didn't fixed it. But with the line #33 it seems to work.

Comment (peterpeterparker) [reply frontend/src/tests/e2e/accounts.spec.ts:33]: Thanks! Added it to the Svelte v4 branch

Comment (dskloetd) [reply frontend/src/tests/e2e/accounts.spec.ts:33]: Please don't do this. Let's find out why this is necessary.

Comment (mstrasinskis) [reply frontend/src/tests/e2e/accounts.spec.ts:33]: Sure, closing the pr 👍

**Pull request closed without merging**